services/src/pro/contexts/postgres.rs

```diff
@@ -533,9 +532,8 @@ where
     fn user_db_ref(&self) -> &Self::UserDB {
         &self.user_db
     }
-
-    fn oidc_request_db(&self) -> &OIDCRequestsDB {
-        &self.oidc_request_db
+    fn oidc_request_db(&self) -> Option<&OidcRequestDb> {
+        self.oidc_request_db.as_ref().as_ref()
```

Comment:
Is this weird?

Reply:
If it is optional to have it, I guess it is fine.